api/v1/api.go

```diff
@@ -447,6 +451,9 @@ func (api *API) insertAlerts(w http.ResponseWriter, r *http.Request, alerts ...*
 		if err := a.Validate(); err != nil {
 			validationErrs.Add(err)
 			api.m.Invalid().Inc()
+			if api.alertLCObserver != nil {
+				api.alertLCObserver.Rejected("Invalid", a)
```

Comment:
can we change the invalid to the actual error?

Reply:
updated